src/components/common/EntityInfo.vue

```diff
@@ -26,7 +26,7 @@
           <v-list-item-subtitle class="business-info">
             <dl>
               <dt>{{ entityTitle() }}</dt>
-              <dd>Name Request No: <span id="entity-nr-number">{{ getBusinessIdentifier }}</span></dd>
+              <dd>Name Request No: <span id="entity-nr-number">{{ getNameRequestNumber }}</span></dd>
```

Comment:
Does this work if there's no NR? (ie, test with Numbered Company)

Reply:
well, it doesn't crash, but it will have no number. If this isn't handled in a future ticket I can add a conditional? Thoughts?

Reply:
Will be handled by  /bcgov/entity#2548 .